models/tmp/stg_iterable__user_unsub_message_type_tmp.sql

```diff
@@ -0,0 +1,14 @@
+{{ config(enabled=var('iterable__using_user_unsubscribed_message_type', True)) }}
+
+{% if does_table_exist('user_unsubscribed_message_type') %}
+
+select * 
+from {{ var('user_unsubscribed_message_type') }}
+
+{% else %}
+
+select * 
+from {{ var('user_unsubscribed_message_type_history') }}
+
+{% endif %}
```

Comment:
We can actually clean this up by doing the following instead of having a full conditional block.

```suggestion
select *
from {{ var('user_unsubscribed_message_type') if does_table_exist('user_unsubscribed_message_type') else var('user_unsubscribed_message_type_history') }}
```

Reply:
Amazing!